packages/web-app/src/components/ErrorHandlingPages/components/ErrorWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+import type { FunctionComponent, ReactElement } from 'react'
+import type { WithStyles } from 'react-jss'
+import withStyles from 'react-jss'
+import type { SaladTheme } from '../../../SaladTheme'
+import ErrorBoundaryImage from '../assets/ErrorBoundaryImage.png'
+
+const styles = (theme: SaladTheme) => ({
```

Comment:
let's use `CSS.Properties` for autocomplete

```suggestion
const styles: (theme: SaladTheme) => Record<string, CSS.Properties> = (theme: SaladTheme) => ({
```

Reply:
Changed